src/components/ArticleDisplay.jsx

```diff
@@ -1,12 +1,17 @@
 import { useEffect, useState } from "react";
-import { getArticleById } from "../utils/api";
+import {
+  addArticleVote,
+  deleteArticleVote,
+  getArticleById,
+} from "../utils/api";
 import Article from "./Article";
 
 const ArticleDisplay = ({ article_id }) => {
   const [article, setArticle] = useState({});
-  //   const [votes, setVotes] = useState(0);
   const [isLoading, setIsLoading] = useState(true);
   const [isError, setIsError] = useState(false);
+  const [isUpVoteError, setIsUpVoteError] = useState(false);
+  const [isDownVoteError, setIsDownVoteError] = useState(false);
```

Comment:
You could most likely get away with one set of error state rather than for up and down voting here. Think of it this way, this one page has 5 sets of state currently, does it really need to?